### ISSUE TITLE
TypeError: Cannot read property 'timestamp' of undefined

### ISSUE THREAD

**Issue by cascading-jox** [helium/mappers#67]
I found a hexagon which when pressed throws the error _TypeError: t[0] is undefined_ on firefox or _TypeError: Cannot read property 'timestamp' of undefined_ on chrome. It is located at _58.405858_, _15.628672_. 

https://user-images.githubusercontent.com/8418570/125160276-56008100-e17c-11eb-910d-c3b03249c66a.mp4

**Comment by waiweng83** [comment]
I noticed that this will happen if the mapper send the altitude value which is not a rounded integer (eg: float).

**Comment by tomtobback** [comment]
yes i'm having the same problem on a few hexagons. i noticed indeed the [doc](https://docs.helium.com/use-the-network/coverage-mapping/mappers-api/) says altitude needs to be `int`, but i assume it's easy for the mapper api to `parseInt()` to fix incoming float values, and i hope the web interface can be fixed to deal with the incorrect data that got into the database?

**Comment by kent-williams** [comment]
Closing this as those hexes now load just fine. This was likely a discrepancy between what was loaded from the tileserver and what could be queried from the database.
